addon/mixins/focusing.js

```diff
@@ -1,21 +1,28 @@
-import Ember from "ember";
-import getOwner from 'ember-getowner-polyfill';
+import Ember from 'ember';
 
-const { get } = Ember;
+/* globals getOwner */
+// One of these two will be uncommented at build time:
```

Comment:
Another option is to just error if `Ember.getOwner` is not present (and the message would instruct the consumer to upgrade Ember (:troll:) or install the polyfill themselves).